app/grandchallenge/blogs/models.py

```diff
@@ -28,6 +36,8 @@ class Post(models.Model):
         variations=settings.STDIMAGE_SOCIAL_VARIATIONS,
     )
 
+    tags = models.ManyToManyField(to=Tag, related_name="blog_tags")
```

Comment:
The related name is how you get these objects (Posts) from the Tags. So, with what you have you would end up with

```
tag = Tag.objects.first()
posts_for_tag = tag.blog_tags.all()
```

Instead, I would suggest `related_name="posts"`, so that `posts_for_tag = tag.posts.all()`.